core/src/main/scala/org/apache/spark/memory/StaticMemoryManager.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.memory
 
+import scala.util.control.NonFatal
+
```

Comment:
by the way I'm not sure whether it's worth it to change anything here. This is for legacy mode and will be removed in the future anyway, and for those people who *are* using it they're doing so because the new workload doesn't work for them, so it's best to just keep things the same here as much as possible. I don't feel super strongly about reverting the changes here but I don't think it's totally necessary.